src/selectors/getNodeInputOptions.js

```diff
@@ -16,6 +16,12 @@ export default (state, nodeId) => {
       label: 'Audio',
       exclude: ['linkableAction'],
     },
+    {
+      value: 'texture',
+      type: 'texture',
+      label: 'Texture',
+      exclude: ['linkableAction'],
```

Comment:
Maybe we want textures to be linkable? A little out of my depth with this system, was mostly just copying audio to get the audio texture into the store

Reply:
A a `linkableAction` is any button that can be connected with inputs. So for instance, the "enable" button for an LFO is linkable. The only inputs that these are enabled for are user controlled MIDI. So you were right to exclude it for audio texture, it definitely wouldn't make much sense!

Reply:
Although I'm a bit confused as to how this can be assigned to a param, is it a 1D texture?